mastodon-terms-phpc.social.rst

```diff
@@ -0,0 +1,60 @@
+=============================================
+Only Safety Act Specific Terms and Conditions
+=============================================
+
+*To implement "ICU G1 Terms of service: substance (all services)" we have
+added the following section into our "about page"
+(https://phpc.social/about).*
+
+Handling Illegal Content
+------------------------
+
+The moderation team takes illegal content extremely seriously. When being made
```

Comment:
Just a suggestion since "extremely seriously" sounds wordy, and there's no need for the word "being" in "When being made aware..."

```suggestion
The moderation team treats illegal content with the utmost gravity. When made
```